indy_node/test/auth_rule/test_get_auth_rule.py

```diff
@@ -143,12 +135,58 @@ def test_get_all_auth_rule_transactions_after_write(looper,
                                                 auth_action=auth_action, auth_type=auth_type,
                                                 field=field, new_value=new_value,
                                                 constraint=constraint)
-    auth_key = ConfigReqHandler.get_auth_key(resp[0][0][OPERATION])
+    str_auth_key = ConfigReqHandler.get_auth_key(resp[0][0][OPERATION])
     resp = sdk_get_auth_rule_request(looper,
                                      sdk_wallet_trustee,
                                      sdk_pool_handle)
-    expect = {key: constraint.as_dict if constraint is not None else {}
-              for key, constraint in auth_map.items()}
-    expect[auth_key] = constraint
+
     result = resp[0][1]["result"][DATA]
-    assert result == expect
+    for rule in result:
+        key = ConfigReqHandler.get_auth_key(rule)
+        if auth_map[key] is None:
+            assert {} == rule[CONSTRAINT]
+        elif key == str_auth_key:
+            assert constraint == rule[CONSTRAINT]
+        else:
+            assert auth_map[key].as_dict == rule[CONSTRAINT]
+
+
+def test_auth_rule_after_get_auth_rule(looper,
+                                       sdk_wallet_trustee,
+                                       sdk_pool_handle):
+    constraint = generate_constraint_list(auth_constraints=[generate_constraint_entity(role=TRUSTEE),
+                                                            generate_constraint_entity(role=STEWARD)])
+    # get all auth rules
+    auth_rules = sdk_get_auth_rule_request(looper,
+                                           sdk_wallet_trustee,
+                                           sdk_pool_handle)
+
+    # prepare action key
+    dict_key = dict(auth_rules[0][1][RESULT][DATA][0])
+    dict_key.pop(CONSTRAINT)
+
+    # prepare "operation" to send AUTH_RULE txn
+    op = dict(auth_rules[0][1][RESULT][DATA][0])
+    op[TXN_TYPE] = AUTH_RULE
+    op[CONSTRAINT] = constraint
```

Comment:
Can we send AUTH_RULE with the same Constraint (that it don't override ` op[CONSTRAINT] `)?

Reply:
Yes. Are we need to add this test?

Reply:
Yes, would be great, to make sure that we can use the GET output as-is for the SET input